debian/snapd.refresh.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=Automatically refresh installed snaps
-After=network.target snapd.socket
-Requisite=snapd.socket
+After=network-online.target snapd.socket
```

Comment:
Sounds good as a default, but I suspect we'll need to add a flag to undo this soon, to support snaps that are themselves part of the network configuration stack.